src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -181,6 +181,31 @@ function validatePropTypes(element) {
   }
 }
 
+/**
+ * Given an element, freeze `type` and `defaultProps` (Only used in DEV)
+ *
+ * @param {ReactElement} element
+ */
+function freezeTypeAndDefaultProps(element) {
```

Comment:
in this case, invocation only needs to be wrapped... since this block will be considered unreachable, `uglifyjs` dead code elimination excludes it from minified production builds